.ci/mount-volume/simple-pod.sh

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env bash
+# First parameter either "Start" or "Stop"
+function podHasStatus {
```

Comment:
I'd consider splitting this function into a start version and a stop version, since the control-flow overlap is almost none. This could also simplify the `$ACTION` checks.

```bash
function podStopped {
  POD_NAME=$1
  if oc get pod "$POD_NAME" -o json > /dev/null 2>&1; then
    # pod found, not deleted yet.
    return 1
  else
    return 0
  fi
}

function podStarted {
  POD_NAME=$1
  SIMPLE_POD_JSON=$(oc get pod "$POD_NAME" -o json)
  RETURN_CODE=$?

  if [[ $RETURN_CODE -ne 0 ]]; then
    echo "Could not obtain information about pod. Trying again."
    return 1
  fi

  POD_STATUS=$(echo "$SIMPLE_POD_JSON" | jq --raw-output '.status.phase')

  echo "Wanted: Running     Actual: ${POD_STATUS}"
  if [[ $POD_STATUS == "Running" ]]; then
    echo "Pod is running."
    return 0
  else 
    return 1
  fi
}
```

Reply:
I've updated the logic of the script to decrease the complexity